activemodel/lib/active_model/errors.rb

```diff
@@ -331,6 +341,18 @@ def add_on_blank(attributes, options = {})
     #
     #   person.errors.add :name, :blank
     #   person.errors.added? :name, :blank # => true
+    #
+    # When used to check for a +message+ that includes an interpolated value
+    # (e.g., +count+), it requires that we specify this value as an option.
```

Comment:
@rafaelfranca I find it unintuitive to need to specify interpolated value during check. Presumably `added?`  will be used in specs a lot, and most times developer just want the granularity to be at message level.

I agree that even if we include this in the docs, developers still won't avoid it unless they encountered it themselves. Maybe the better way to do this is provide another method which can offer this level of granularity.